extensions/panache/hibernate-reactive-rest-data-panache/deployment/src/test/java/io/quarkus/hibernate/reactive/rest/data/panache/deployment/entity/CollectionsResource.java

```diff
@@ -1,8 +1,32 @@
 package io.quarkus.hibernate.reactive.rest.data.panache.deployment.entity;
 
+import java.util.Collections;
+
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.POST;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.PathParam;
+
+import io.quarkus.hibernate.reactive.panache.common.WithSessionOnDemand;
 import io.quarkus.hibernate.reactive.rest.data.panache.PanacheEntityResource;
 import io.quarkus.rest.data.panache.ResourceProperties;
+import io.smallrye.mutiny.Uni;
 
 @ResourceProperties(hal = true, paged = false, halCollectionName = "item-collections")
+@WithSessionOnDemand
```

Comment:
This annotation should not be necessary here, or?

Reply:
This was actually a leftover, thanks for spotting!

Reply:
PR updated